### PR TITLE
feat: add gemini code sample and test for text-only input

### PR DESCRIPTION
**Pull request by irataxy**
## Description

Fixes b:336785030

- [X] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/gemini_text_input_example.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    # [START generativeaionvertexai_gemini_generate_from_text_input]
+    import vertexai
+    from vertexai.generative_models import GenerativeModel, Part
+
+    # TODO(developer): Update and un-comment below line
+    # project_id = "PROJECT_ID"
+
+    vertexai.init(project=project_id, location="us-central1")
+
+    model = GenerativeModel(model_name="gemini-1.0-pro")
```
Comment (irataxy) [inline generative_ai/gemini_text_input_example.py:29]: Is this the correct model? Does it need to have the form gemini-1.0-pro-XXX?

Comment (holtskinner) [inline generative_ai/gemini_text_input_example.py:29]:
```suggestion
    model = GenerativeModel(model_name="gemini-1.0-pro-002")
```

File: generative_ai/gemini_text_input_example_test.py
Comment (holtskinner) [inline generative_ai/gemini_text_input_example_test.py:1]: Can you add this test to https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/generative_ai/test_gemini_examples.py

**Review by holtskinner — APPROVED** [review]

File: generative_ai/gemini_text_input_example_test.py
Comment (irataxy) [reply generative_ai/gemini_text_input_example_test.py:1]: Done

**Pull request merged**